zip-0000.rst

```diff
@@ -623,27 +637,46 @@ Consensus ZIP, a Deployment section MUST be present in order for the ZIP
 to change status to Proposed. Typically, although not necessarily, this
 will specify a network upgrade in which the consensus change is to activate.
 
-A Standards ZIP SHOULD only change status from Proposed to Implemented once
-the Owners provide an associated reference implementation, typically in
-the period after the network upgrade's specification freeze but before
+A ZIP's status is Released if it is Proposed, Active, Implemented, or Final
+(i.e. not Draft, Rejected, Obsolete, or Withdrawn).
+
+A ZIP SHOULD NOT be changed from a non-Released status to a Released
+status if there is significant community opposition to its content.
```

Comment:
How to evaluate "significant community opposition to its content"?

Reply:
Okay, back to 'How to evaluate "significant community opposition to its content"?'.

Basically this is at the discretion of the Editors. There will be some cases where it's obvious that there is "significant community opposition", and some cases where it's a grey area. As an example where it was obvious, I'd cite the locking of mining rewards discussed in [this forum thread](https://forum.zcashcommunity.com/t/releasing-mining-rewards-over-50-weeks/32363). (That never got to the point of being even a Draft ZIP, but if it had with the opposition to it still being as strident and persistent, then that opposition could have been reason to block the ZIP moving to a Released status. Disclosure of interest: [I opposed it](https://forum.zcashcommunity.com/t/releasing-mining-rewards-over-50-weeks/32363/21).)

In any case, note that this whole list of rejection reasons is about what the Editors MAY do:

> The ZIP Editors MAY reject a new ZIP or an update to an existing ZIP, by consensus among the current Editors. Rejections can be based on any of the following reasons: [...]

I think it's unnecessary to resolve all ambiguity in wording on a condition that is only a MAY in any case. If we thought that ZIP Editors should be completely unable to reject new ZIPs or ZIP updates citing community opposition, then it might be worth being more careful. But I'm pretty sure this is a good reason for rejection in at least some cases. Rejection isn't a permanent decision anyway; the Owners could always try to lobby the Editors to change their minds (or more productively, try to address the community opposition).